tfjs-tfdf/src/tfdf_web_api_client.d.ts

```diff
@@ -19,4 +19,8 @@ import {TFDFWebAPIClient} from './types/tfdf_web_api';
 /**
  * The main export for tfdf_web_api_client.js types.
  */
-export declare let tfdfWeb: Promise<TFDFWebAPIClient>;
+export declare let tfdfWeb: () => Promise<TFDFWebAPIClient>;
+
+type LocateFileFunction = (path: string, prefix?: string) => string;
+/** The global function to set WASM path. */
+export declare let setLocateFile: (locateFile: LocateFileFunction) => void;
```

Comment:
Tiny nit: I think this can be const, although I'm not actually that knowledgeable about .d.ts files.
```suggestion
export declare const setLocateFile: (locateFile: LocateFileFunction) => void;
```